src/Goteo/Library/Forms/Admin/AdminWorkshopEditForm.php

```diff
@@ -74,111 +78,114 @@ public function createForm() {
                 'pre_addon' => '<i class="fa fa-globe"></i>',
                 'attr' => ['help' => Text::get('tooltip-project-project_location')]
             ])
-           ->add('date_in', 'datepicker', array(
+            ->add('date_in', DatepickerType::class, [
                 'label' => 'admin-title-date-in',
                 'required' => true,
-                'constraints' => array(new Constraints\NotBlank()),
-            ))
-           ->add('date_out', 'datepicker', array(
+                'constraints' => [new Constraints\NotBlank()],
+            ])
+            ->add('date_out', DatepickerType::class, [
                 'label' => 'admin-title-date-out',
                 'required' => false,
-                'constraints' => array(new Constraints\NotBlank()),
-            ))
-            ->add('schedule', 'text', [
+                'constraints' => [new Constraints\NotBlank()],
+               ])
+            ->add('schedule', TextType::class, [
                 'label' => 'regular-schedule',
-                'required' => false,
+                'required' => true,
```

Comment:
Is it required???

Reply:
In the database, the field is NOT NULL. I remember that I couldn't save the form without making this change.